src/sagemaker_containers/environment.py

```diff
@@ -188,43 +202,19 @@ def cpu_count():  # type: () -> int
 
 
 class Environment(collections.Mapping):
-    """Provides access to aspects of the training environment relevant to training jobs, including
-    hyperparameters, system characteristics, filesystem locations, environment variables and configuration settings.
+    """Base Class which provides access to aspects of the environment including
+    system characteristics, filesystem locations, environment variables and configuration settings.
 
     The environment is a read-only snapshot of the container environment. It does not contain any form of state.
     It is a dictionary like object, allowing any builtin function that works with dictionary.
 
-    Example on how to print the state of the container:
-        >>> print(str(smc.Environment.create()))
-
-    Example on how a script can use training environment:
-        ```
-        >>>import sagemaker_containers as smc
-        >>>env = smc.Environment.create()
-
-        get the path of the channel 'training' from the inputdataconfig.json file
-        >>>training_dir = env.channel_input_dirs['training']
-
-        get a the hyperparameter 'training_data_file' from hyperparameters.json file
-        >>>file_name = env.hyperparameters['training_data_file']
-
-        # get the folder where the model should be saved
-        >>>model_dir = env.model_dir
-
-        >>>data = np.load(os.path.join(training_dir, training_data_file))
-
-        >>>x_train, y_train = data['features'], keras.utils.to_categorical(data['labels'])
-
-        >>>model = ResNet50(weights='imagenet')
-
-        ...
-        >>>model.fit(x_train, y_train)
-
-        save the model in the end of training
-        >>>model.save(os.path.join(model_dir, 'saved_model'))
-        ```
+    Attributes:
+            current_host (str): The name of the current container on the container network. For example, 'algo-1'.
+            num_gpu (int): The number of gpus available in the current container.
+            num_cpu (int): The number of cpus available in the current container.
+            module_name (str): The name of the user provided module.
+            module_dir (str): The full path location of the user provided module.
```

Comment:
Do we need these attributes there? I guess I added by mistake. Should we remove?

Reply:
No, these are the common attributes shared by Serving and Training.